src/components/SignUp/SignUp.js

```diff
@@ -56,14 +56,9 @@ class SignUp extends React.Component {
         email,
         password,
         phone
-      })
-        .then(response => {
-          console.log("success" + JSON.stringify(response));
-        })
-        .catch(error => {
-          console.log("error" + error);
-        });
-      console.log(response);
+      });
+
+      console.log(response.err);
```

Comment:
open notification

Reply:
1) create reusable logic for login function
2) login function should return user
3) check if user.emailVerified === true ? redirect ot main page :  redirect to email not verified page

Reply:
![image](https://user-images.githubusercontent.com/19416734/72900006-cc62b680-3d2f-11ea-8b67-68b8313ff214.png)
![image](https://user-images.githubusercontent.com/19416734/72900030-d5ec1e80-3d2f-11ea-9ed0-a7ed5433e219.png)

I have no ideas how to handle errors?!